ci-operator/step-registry/hypershift/openstack/create/hypershift-openstack-create-chain.yaml

```diff
@@ -0,0 +1,208 @@
+chain:
+  as: hypershift-openstack-create
+  steps:
+  - as: create-hostedcluster
+    cli: latest
+    env:
+    - name: HYPERSHIFT_NODE_COUNT
+      default: "3"
+      documentation: "The number nodes to automatically create and join to the cluster."
+    - name: HYPERSHIFT_BASE_DOMAIN
+      default: "origin-ci-int-aws.dev.rhcloud.com"
+      documentation: "The cluster's FQDN will be a subdomain of the base domain."
+    - name: HYPERSHIFT_AWS_REGION
+      default: "us-east-1"
+      documentation: |
+        Specifies the AWS region for the cluster. If intentionally left as an empty string, 
+        the region defaults to that of the management cluster.
+    - name: HYPERSHIFT_HC_ZONES
+      default: "us-east-1a"
+      documentation: |
+        Specifies the AWS AZs to create the NodePools. If intentionally left as an empty string,  
+        it defaults to an AZ utilized by the management cluster.
+    - name: EXTRA_ARGS
+      default: ""
+      documentation: "Extra args to pass to the create cluster openstack command"
+    - name: ENABLE_ICSP
+      default: "false"
+      documentation: "if true, add image content sources config(path=${SHARED_DIR}/mgmt_icsp.yaml)"
+    - name: HYPERSHIFT_HC_RELEASE_IMAGE
+      default: ""
+      documentation: "Release image used for the HostedCluster. Empty by default it will use release:latest"
+    - name: HYPERSHIFT_CP_AVAILABILITY_POLICY
+      default: "SingleReplica"
+      documentation: "Availability policy for hosted cluster components. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+    - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
+      default: "SingleReplica"
+      documentation: "Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+    - name: HYPERSHIFT_CREATE_CLUSTER_RENDER
+      default: "false"
+      documentation: "If true, render artifacts to ${SHARED_DIR}/hypershift_create_cluster_render.yaml"
+    - default: "rhcos-latest-hcp-nodepool"
+      name: RHCOS_IMAGE_NAME
+    commands: |-
+      set -exuo pipefail
```

Comment:
I am afraid of `-x` in the CI tbh. I know it's a pain, but have you considered logging instead?

Reply:
I stole this script from here:

https://github.com/openshift/release/blob/1771d8a973f464bdf44375c8ed392c5dff372b71/ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml#L54

However I agree with you and I think in this case we should be fine with it it. I'll drop it.